packages/view/src/Charcoal/View/Twig/DebugHelpers.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Charcoal\View\Twig;
+
+// From 'charcoal-view'
+use Charcoal\View\Twig\HelpersInterface;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * Twig helpers for Url.
+ */
+class DebugHelpers extends AbstractExtension
+    implements HelpersInterface
+{
+    /**
+     * @param array $data Class Dependencies.
+     */
```

Comment:
Remove incorrect and extraneous block comments, and `implements` should be on the same line as `extends`.

```suggestion
class DebugHelpers extends AbstractExtension implements
    HelpersInterface
{
```